src/applications/caregivers/utils/mapbox/mapboxToken.js

```diff
@@ -1,3 +1,3 @@
 export const mapboxToken =
   (typeof process !== 'undefined' && process.env.MAPBOX_TOKEN) ||
-  'pk.eyJ1IjoibWtlcm5zMDEiLCJhIjoiY2xuMjVxb2ZoMDN4bDJybm8zNWYxdXR6NiJ9.Kx5vRDlNOF6-c9VP5eAEyg';
+  'pk.eyJ1IjoiYWRob2MiLCJhIjoiY2wyZjNwM3dxMDZ4YjNjbzVwbTZ5aWQ1dyJ9.D8TZ1a4WobqcdYLWntXV_w';
```

Comment:
This id was failing for me locally. It looks like the prod api key is expected to be exposed, so we want that as the default. https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/facility-locator/engineering/data-and-api-integrations/mapbox/mapbox-implementation.md

Reply:
> contrary to what we're likely conditioned to think, our prod API key is not sensitive information but our dev key is sensitive

yes that is contrary to what I expected! okay, I guess???

Reply:
haha that was my exact response.